src/20240829_Multi_StablecoinIRCurveAmendment/AaveV2Ethereum_StablecoinIRCurveAmendment_20240829.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV2PayloadEthereum} from 'aave-helpers/src/v2-config-engine/AaveV2PayloadEthereum.sol';
+import {EngineFlags} from 'aave-v3-periphery/contracts/v3-config-engine/EngineFlags.sol';
+import {IAaveV2ConfigEngine} from 'aave-helpers/src/v2-config-engine/IAaveV2ConfigEngine.sol';
+import {IV2RateStrategyFactory} from 'aave-helpers/src/v2-config-engine/IV2RateStrategyFactory.sol';
+/**
+ * @title Stablecoin IR Curve Amendment
+ * @author Aave Chan Initiative
+ * - Snapshot: Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/arfc-chaos-labs-stablecoin-ir-curve-amendment-on-aave-v2-and-v3-2024-08-15/18669
+ */
+contract AaveV2Ethereum_StablecoinIRCurveAmendment_20240829 is AaveV2PayloadEthereum {
+  function rateStrategiesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV2ConfigEngine.RateStrategyUpdate[] memory)
+  {
+    IAaveV2ConfigEngine.RateStrategyUpdate[]
+      memory rateStrategies = new IAaveV2ConfigEngine.RateStrategyUpdate[](3);
+    rateStrategies[0] = IAaveV2ConfigEngine.RateStrategyUpdate({
+      asset: AaveV2EthereumAssets.USDT_UNDERLYING,
+      params: IV2RateStrategyFactory.RateStrategyParams({
+        optimalUtilizationRate: EngineFlags.KEEP_CURRENT,
+        baseVariableBorrowRate: EngineFlags.KEEP_CURRENT,
+        variableRateSlope1: _bpsToRay(5_50),
```

Comment:
@Rozengarden any reason to reduce rates on Aave V2 Ethereum and not on Polygon and Avax V2 Markets?

Reply:
From CL: mainnet tvl is bigger and the polygon and avalanche market were addressed separately: https://app.aave.com/governance/v3/proposal/?proposalId=154